src/sentry/models/group.py

```diff
@@ -803,6 +804,23 @@ def get_recommended_event_for_environments(
             else self.get_latest_event_for_environments([env.name for env in environments])
         )
 
+    def get_suspect_commit(self) -> Commit | None:
```

Comment:
Nice addition, thank you! Wish there was a higher level we could keep this at, but a good start